docs/undergrad.yaml

```diff
@@ -356,7 +356,8 @@ Single Variable Real Analysis:
     Weierstrass trigonometric approximation theorem: 'span_fourier_closure_eq_top'
   Convexity:
     convex functions of a real variable: 'ConvexOn'
-    continuity and differentiability of convex functions: 'https://en.wikipedia.org/wiki/Convex_function#Functions_of_one_variable'
+    continuity of convex functions: 'ConvexOn.continuousOn'
+    differentiability of convex functions: 'https://en.wikipedia.org/wiki/Convex_function#Functions_of_one_variable'
```

Comment:
@PatrickMassot, just checking: Is it fine if I split this item in two?

Reply:
my impression is that we really want a bijection between undergrad.yaml and the entries in the corresponding PDF sanctioned by the French government (or whatever).

So it would be find to include this in one of the other yaml's, but it will have to wait a bit before this one can be ticked off.

@PatrickMassot feel free to correct me if I'm wrong.

Reply:
We've performened splits in the past - see "power series expansion of elementary functions" below, which gained sub-items.

Reply:
I have done so